### PR TITLE
Verification of TFS

### PR DESCRIPTION
**Pull request by Gee19**
* Remove useless password help text from base plugin
* Add area path & collection name fields
* Add advanced options section to system level
* Add missing scope to TFS fields
* Append tfs_collection field to connection details section
* Show area path & iteration path fields on project level under
Advanced TFS Configuration section.

Refs #SDE-3970

### CODE REVIEW

File: sdetools/modules/sync_tfs/tfs_metadata.py [hunk 1]
```diff
@@ -25,43 +25,74 @@ def supports_references(self):
     def __init__(self, phases):
         super(TfsMetadata, self).__init__(phases)
 
+        self.alm_server.placeholder = 'bugs.server.com'
+
```
Comment (lmontopo) [inline sdetools/modules/sync_tfs/tfs_metadata.py]: whats the decision here @simon? Are we going with "bugs.server.com" whenever there isn't an alm specific placeholder to put? Or are we leaving it blank in such cases?

File: sdetools/modules/sync_tfs/tfs_metadata.py [hunk 2]
```diff
@@ -25,43 +25,74 @@ def supports_references(self):
...
             label=_('States which map to DONE in SD Elements'),
             field_type=FieldType.STRING,
-            placeholder='Done',
-            default='Done'
+            default='Done',
+            scope=['system']
         )
         self.tfs_new_status = IntegrationField(
             name='tfs_new_status',
             label=_('Open a story in %s with the following status' % self.name),
```
Comment (lmontopo) [inline sdetools/modules/sync_tfs/tfs_metadata.py]: Our wording should be consistent.  Here we are using "open a _story_", but below (for the reopen status) we use "reopen a _work item_".  I don't have any preference between _work item_ and _story_ i just think we should pick one and use it for both fields.  Thoughts?

**Comment by lmontopo** [comment]
Everything looks good, just a couple small questions / suggestions.

File: sdetools/modules/sync_tfs/tfs_metadata.py (hunk 2 above)
Comment (Gee19) [inline sdetools/modules/sync_tfs/tfs_metadata.py]: I definitely think we should use the proper terminology for the specific ALM. Fixing now

Comment (lmontopo) [inline sdetools/modules/sync_tfs/tfs_metadata.py]: what is the property terminology? work item or story?

File: sdetools/modules/sync_tfs/tfs_metadata.py (hunk 1 above)
Comment (Gee19) [inline sdetools/modules/sync_tfs/tfs_metadata.py]: Removed it, it was set as a placeholder in the base class.

**Pull request merged**